src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/ua/" class="logo"></a>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">samsung</a>
          </li>
```
add empty blanks between siblings for better readability